CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(fdict
   DESCRIPTION "Fortran dictionary for arbitrary data-types"
 )
 
+if (NOT DEFINED CMAKE_Fortran_MODULE_DIRECTORY)
+	set(CMAKE_Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/fortran_modules)
```

Comment:
add quotation, to ensure paths with spaces

Reply:
I fully agree. This is a relic of the original cmake design, and although there are rumors that the lead devs are working on a more robust, non text-based interface for camke 4, these are here to stay :(.

I was doing the same thing until I had various things break down because of the quotation. I realized though that most cmake commands are designed to work on variables (e.g. the `if()` command), so I work on the assumption that most commands are designed and tested to work for unquoted formats, and just use the quotation when it is needed to enforce it.